src/main/java/com/radixdlt/crypto/ECKeyPair.java

```diff
@@ -244,4 +244,12 @@ public EncryptedPrivateKey encryptPrivateKeyWithPublicKey(ECPublicKey publicKeyU
 			throw new IllegalStateException("Failed to encrypt `this.privateKey` with provided `ECPublicKey`", e);
 		}
 	}
+
+	public String publicKeyHex() {
+		return Bytes.toHexString(getPublicKey().getBytes());
```

Comment:
This method should be in ECPublicKey, rather than here.

Reply:
Sugar for tests, relating to my [comment above](https://github.com/radixdlt/radixdlt-java-common/pull/13#discussion_r451418622)

Reply:
OK, so if it's only for tests, then make it package-local (remove the public visibility) and annotate with VisibleForTesting.